Sources/StreamChatUI/Utils/TextViewMentionedUsersHandler.swift

```diff
@@ -23,7 +23,7 @@ class TextViewMentionedUsersHandler {
         else {
             return nil
         }
-        let name = String(text[range].replacingOccurrences(of: "@", with: ""))
+        let name = String(text[range].dropFirst())
```

Comment:
i know this is an implementation detail on our end-- but we actually override and hide the @ symbol in the message. by calling `dropFirst()` youll remove whatever the first letter is... Could you maybe only dropFirst if its the @mention delimiter (@)?

Reply:
Nice catch 👍 I've opted for another approach which should work in every case 👍